scripts/release/mule/test/test.sh

```diff
@@ -10,35 +11,52 @@ export ARCH_TYPE
 OS_TYPE=$(./scripts/ostype.sh)
 export OS_TYPE
 
-if [ -z "$VERSION" ]; then
-    VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
-fi
-export VERSION
-
-if [ -z "$BRANCH" ]; then
-    BRANCH=${BRANCH:-$(git rev-parse --abbrev-ref HEAD)}
-fi
-export BRANCH
+export BRANCH=${BRANCH:-$(git rev-parse --abbrev-ref HEAD)}
+export CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
+export NETWORK=${NETWORK:-$(./scripts/compute_branch_network.sh "$BRANCH")}
+export SHA=${SHA:-$(git rev-parse HEAD)}
+export VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
 
-if [ -z "$CHANNEL" ]; then
-    CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
-fi
-export CHANNEL
+PKG_DIR="./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
 
-if [ -z "$SHA" ]; then
-    SHA=${SHA:-$(git rev-parse HEAD)}
-fi
-export SHA
+pushd "$PKG_DIR"
 
 if ! $USE_CACHE
 then
-    mule -f package-test.yaml "package-test-setup-$PKG_TYPE"
+    SRC_DIR="s3://algorand-staging/releases/$CHANNEL/$VERSION"
+
+    # deb
+    aws s3 cp "$SRC_DIR/algorand_${CHANNEL}_${OS_TYPE}-${ARCH_TYPE}_${VERSION}.deb" .
```

Comment:
should these copy commands be conditional based on the OS_TYPE and ARCH_TYPE?

Reply:
I'm not sure I understand the question.  The naming convention for the packages are pretty firmly set, and if there are different os types and arch types this should just work.

Reply:
Are deb and rpm packaging appropriate for all combinations of OS and ARCH?